### ISSUE TITLE
Feature Request: option to specify frame rate of animation

### ISSUE THREAD

**Issue by stefanuddenberg** [paperjs/paper.js#1581]
Although the [Paper.js docs](http://paperjs.org/tutorials/animation/creating-animations/) say that the "onFrame" event is called up to 60 times per second, I noticed that animations go much faster on monitors with higher refresh rates, presumably since it's relying on "requestAnimationFrame". Since more and more people are using high refresh-rate monitors, it would be nice to be able to specify the frame rate, so that every user gets a consistent experience. Right now I'm resorting to some cludgy work-arounds for such monitors (e.g., checking if the time elapsed since the last frame request is large enough)...

**Comment by lehni** [comment]
Yes we probably should add this now. Here's a starting point: https://stackoverflow.com/a/19772220

**Comment by christopherball** [comment]
Is this request going to be addressed by chance?  For example, I have a 100hz monitor, and the nearest divisible whole number <= to the hard-coded 60 fps cap is 50 in my case.  This means I can't exceed 50fps no matter the performance of my system or efficiency of my code.
